src/SFA.DAS.Charities.Database/Tables/Charity.sql

```diff
@@ -0,0 +1,24 @@
+﻿CREATE TABLE [dbo].[Charity]
+(
+    [Id] INT NOT NULL PRIMARY KEY, 
+    [Name] VARCHAR(MAX) NOT NULL, 
+    [CompanyNumber] VARCHAR(MAX) NOT NULL, 
+    [RegisteredCharityNumber] INT NOT NULL, 
+    [LinkedCharityId] INT NOT NULL, 
+    [RegistrationDate] DATETIME2 NOT NULL, 
+    [RegistrationStatus] TINYINT NOT NULL, 
+    [Type] TINYINT NULL, 
+    [FinancialPeriodStartDate] DATETIME2 NULL, 
+    [FinancialPeriodEndDate] DATETIME2 NULL, 
+    [Address1] VARCHAR(MAX) NULL, 
+    [Address2] VARCHAR(MAX) NULL, 
+    [Address3] VARCHAR(MAX) NULL, 
+    [Address4] VARCHAR(MAX) NULL, 
+    [Address5] VARCHAR(MAX) NULL, 
```

Comment:
Just a observation as Confluence said to use these but I initially saw these as 5 different addresses rather than AddressLine1, AddressLine2, etc